### ISSUE TITLE
Dissabilities

### ISSUE THREAD

**Issue by carlosbuitragosan** [carlosbuitragosan/wordle#2]
## As a person with a physical disability I should be able to:

- Input a letter to complete a word and submit my guess
- Use the keyboard to submit my guess
- Use a screen reader to submit my guess

**Comment by carlosbuitragosan** [comment]
You can now use the arrow keys to navigate through the on-screen keyboard and the enter key to submit your character
